artemis-server/src/test/resources/ConfigurationTest-full-config.xml

```diff
@@ -57,6 +57,10 @@
       <global-max-size>1234567</global-max-size>
       <max-disk-usage>37</max-disk-usage>
       <disk-scan-period>123</disk-scan-period>
+      <analyze-critical-halt>true</analyze-critical-halt>
```

Comment:
between documentation you name this Critical Analysis, then in config etc you switch round the words to analyse critical. suggest keeping naming constant.